docs/requirements.txt

```diff
@@ -1,9 +1,7 @@
 sphinx
 sphinx_rtd_theme
-numpy>=1.7
 contextlib2
-cloudpickle>=0.3.1
 graphviz>=0.8
-observations>=0.1.4
```

Comment:
i think this is still used in the AIR example from #500
@neerajprad should we move off this dependency ?

Reply:
You are right, it seems that I miss that tutorial while using "grep".

Reply:
This is indeed needed by a few tutorials, let us reinstate it back.